near-plugins-derive/src/access_controllable.rs

```diff
@@ -53,13 +53,16 @@ pub fn access_controllable(attrs: TokenStream, item: TokenStream) -> TokenStream
         #[borsh(crate = "near_sdk::borsh")]
         struct #acl_type {
             /// Stores permissions per account.
+            #[allow(deprecated)]
```

Comment:
I think it would be helpful to add a short comment on `#acl_type` that all `store` types which make sense here have been deprecated. Otherwise future readers might wonder about this.

It seems unfortunate that all `store` types which are iterable have been deprecated by `near-sdk`. Perhaps [TreeMap](https://docs.rs/near-sdk/latest/near_sdk/store/index.html) will be stabilized, but we don't need ordered keys here and shouldn't pay the cost for them.

Reply:
Done